Sources/Services/AWSPolly/PollyClient.swift

```diff
@@ -481,3 +481,21 @@ extension PollyClient: PollyClientProtocol {
     }
 
 }
+
+extension PollyClient {
+    public func getPresignedURLForSynthesizeSpeech(input: SynthesizeSpeechInput, expiration: Foundation.TimeInterval) async throws -> Foundation.URL {
+        guard let presignedURL = try await input.presignURL(config: config, expiration: expiration) else {
+            throw ClientError.unknownError("Returned URL from input.presignURL() was nil for the operation SynthesizeSpeech.")
+        }
+        return presignedURL
+    }
+}
+
+extension PollyClient {
+    public func presignRequestForSynthesizeSpeech(input: SynthesizeSpeechInput, expiration: Foundation.TimeInterval) async throws -> URLRequest {
```

Comment:
Why do URL & request use different language in the method name? They should be consistent with one another:
`presignRequestForSynthesizeSpeech` and `presignURLForSynthesizeSpeech`

Reply:
Changed as suggested.